config/locales/server.en.yml

```diff
@@ -9,6 +9,8 @@ en:
     show_filter_by_solved_status: "Show a dropdown to filter a topic list by solved status."
     notify_on_staff_accept_solved: "Send notification to the topic creator when a post is marked as solution by a staff."
     disable_solved_education_message: "Disable education message for solved topics."
+    accept_solutions_topic_starter: "Allow the topic starter to accept a solution."
```

Comment:
I know this is annoying but could we use "author" instead of "starter?" It's more in line with other language Discourse uses.

Reply:
Yes, author is better. Fixed.